tests/test_run_length.py

```diff
@@ -3,41 +3,60 @@
 import xarray as xr
 import pandas as pd
 import numpy as np
+import os
```

Comment:
I think we're using `Pathlib` in our testing code. We may as well use it consistently. 
```suggestion
from pathlib import Path
```